builtin/mainmenu/dlg_clients_list.lua

```diff
@@ -0,0 +1,59 @@
+--Luanti
+--Copyright (C) 2024 siliconsniffer
+--
+--This program is free software; you can redistribute it and/or modify
```

Comment:
Suggestion: Use an SPDX-License-Identifier line to save some boilerplate.

Reply:
Would leave it like this for now for coherence with the other dialogs.

Reply:
There is no need for consistency with legacy code in different files; in fact if anything I would consider it a (minor) oversight that we haven't gotten rid of this boilerplate in the legacy code yet.